ai_bridge/models/ai_bridge_execution.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import requests
+
+from odoo import _, fields, models
+
+
+class AiBridgeExecution(models.Model):
+
+    _name = "ai.bridge.execution"
+    _description = "Ai Execution"  # TODO
+
+    ai_bridge_id = fields.Many2one(
+        "ai.bridge",
+        required=True,
+        ondelete="cascade",
+    )
+    res_id = fields.Integer(required=True)
+    model_id = fields.Many2one(
```

Comment:
Do you find it necessary to delete the executions of a deleted model?

Reply:
Otherwise it cannot be required....